src/MainWindow.cpp

```diff
@@ -232,7 +233,9 @@ void MainWindow::connectActions()
     // "Treemap" menu
 
     connect( _ui->actionShowTreemap, SIGNAL( toggled( bool )   ),
-	     this,		     SLOT  ( showTreemapView() ) );
+         this,		     SLOT  ( showTreemapView() ) );
```

Comment:
Indentation is now mangled. Please see file Contributing.md about the project's coding standards.